spec/classroom_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe Classroom do
+  let(:classroom) { Classroom.new('A101') }
+
+  describe '#initialize' do
+    it 'should set label correctly' do
+      expect(classroom.label).to eq('A101')
+    end
+
+    it 'should initialize students as an empty array' do
+      expect(classroom.students).to be_empty
+    end
+  end
+
+  describe '#classroom=' do
+    it 'should associate the classroom with the student' do
+      student = double('Student')
+      expect(student).to receive(:classroom=).with(classroom)
+      classroom.add_students(student)
+    end
```

Comment:
- [ ] @Team Great work adding tests for the classroom class, however you forgot to add test for this method  ` classroom=`, its  project requirement to add tests for all methods
![image](https://github.com/Omwomo/school-library/assets/61408860/a1e0a748-ab1a-40c5-b631-9873c106c7df)